content/reference/sql/supported-clients/sql-editors.md

```diff
@@ -7,40 +7,101 @@ title: SQL Editors
 Dolt comes with a built in MySQL compatible server that you can connect SQL Editors too. Here are a list of notable MySQL Editors and our compatibility status.
```

Comment:
change "too" to "to" here, or... better yet... reword this sentence to something like:
"Dolt comes with a built-in MySQL compatible server, making it easy to connect to your Dolt databases with existing SQL tooling." 

That reads a little better and avoids the preposition at the end of the sentence.